### DISCUSSION TITLE
Reheating after print

### DISCUSSION THREAD

**Discussion by rollyceballos** [Jyers/Marlin#594] [Q&A] [answered]
I'm not used to the post-cancel print step of Reheating to 180 for the hotend.  Where is this setting in the Config? Sorry if I didn't see this in the Wiki I've been trying to figure out why it's doing that.  Sometimes I'm in a loop where it times out and I pick a print from the SD card and it goes through Reheating to 180 again.  I have to powercycle the printer to make it work again.

**Comment by DeNohvo** [comment]
Yes i get this to, its quite annoying. If i stop a print the screen sometimes freezes the nozzle temp rises to 180C, if you leave it alone it eventually starts to cool down after reaching 180.

**Reply by rollyceballos** [reply]
Yep, somehow if you leave it for a few minutes it'll 'reset' itself to cooldown state.  Not very helpful when you're keeping the nozzle and bed temp near your target print's settings to print soon, rather than doing a long cooldown cycle.

**Reply by DeNohvo** [reply]
This didn't happen with previous firmware, wonder if its a bug?

**Reply by rollyceballos** [reply]
I only started with Jyer's firmware from 1.3.0 so I've seen it ever since - using 1.3.1 now and it's still there.

**Comment by Jyers** [comment, marked as answer]
So you're saying its reheating to 180 after you cancel a print?

**Reply by rollyceballos** [reply]
yes, and it's doing so as a post-step.  sometimes it timed-out and shows a prompt to 'Continue' shows the 'idle' menu again where I could select a print but I see it targets nozzle at 180c.  Here's where I don't know how it thinks it timed-out.  When I see it reach 180c sometimes I go ahead to print a new file since it reaches 180c and I'm not expecting any other prompt that it finished 'Reheating'...I'm not sure why it does so in the first place.

That's where I get stuck in a loop where it's showing the print state menu but complains Reheating failed.

I'll try to make sure to take a photo of it next time if this is not an expected behavior.

**Reply by Jyers** [reply]
Interesting, I'll look into this.

**Reply by Jyers** [reply]
I you do see this again, open an issue on it and I'll take a better look into what might be happening.

**Reply by rollyceballos** [reply]
Will do, and post some photos too for reference.  thanks @Jyers !
